firefox_ui_tests/remote/security/test_mixed_content.py

```diff
@@ -0,0 +1,40 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+from marionette_driver import Wait
+
+
+class TestMixedContent(FirefoxTestCase):
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.url = 'https://mozqa.com/data/firefox/security/mixedcontent.html'
+
+    def tearDown(self):
+        try:
+            if self.browser.navbar.locationbar.identity_popup.is_open:
+                self.browser.navbar.locationbar.identity_popup.close(force=True)
+        finally:
+            FirefoxTestCase.tearDown(self)
+
+    def test_mixed_content(self):
+        with self.marionette.using_context('content'):
+            self.marionette.navigate(self.url)
+
+        favicon = self.browser.navbar.locationbar.favicon
+        Wait(self.marionette).until(lambda _: favicon.get_attribute('hidden') != 'none')
```

Comment:
This line looks pretty different from what was here before and what's in the mozmill test. Does the favicon have a 'hidden' property? I don't see it in the browser inspector.